assets/css/_sass/structures/_grid.scss

```diff
@@ -0,0 +1,72 @@
+:root {
+  --auto-grid-min-size: 17rem;
+  --l-gap: 1rem;
+}
+
```

Comment:
I think it might make sense to move these into `base/_custom-properties.scss` and import it as one of the first partials, to ensure custom properties are in one place and available throughout the whole CSS